app/interactors/create_article_interactor.rb

```diff
@@ -0,0 +1,15 @@
+module CreateArticleInteractor
+  class Create
```

Comment:
Лучше не использовать interactor в названии, предлагаю все объекты связанные с article помещать в папку interactors/articles 

```suggestion
module Articles
  class Create
```